PWGHF/TableProducer/candidateCreatorBplus.cxx

```diff
@@ -331,10 +349,9 @@ struct HfCandidateCreatorBplusExpressions {
 
   void processMc(aod::HfCand2Prong const& dzero,
                  aod::TracksWMc const& tracks,
-                 aod::McParticles const& mcParticles)
+                 aod::McParticles const& mcParticles,
+                 aod::HfCandBplusProngs const& candsBplus)
```

Comment:
Linked to the comment above, I'm just wondering if you decided to implement this because it behaves better than `rowCandidateBPlus` (better CPU performance, code readability, etc.)

Reply:
As, in this `processMc`, we only want to access the prongs of the B0 candidate, the information we need is in the `HfCandBplusProngs` table only and not in `HfCandBplusExt` table spawned just above, so we need to subscribe to this "prongs table".

Reply:
ok the point is that you you need only indices, therefore no need to query the full table with topological variables. Thank you.